docs/en/04_Changelogs/3.7.0.md

```diff
@@ -0,0 +1,29 @@
+# 3.7.0 (unreleased)
+
+## SilverStripe 3.7 and PHP 7.2 and Object subclasses
+
+SilverStripe 3.7 now supports PHP 7.2, which is exciting, but PHP 7.2 introduces an `object` keyword.
+On a custom project, you can simply replace any uses of `Object` with `SS_Object`.
+
+```diff
+-class MyClass extends Object
++class MyClass extends SS_Object
+{
+	public function myFunction()
+	{
+-		$foo = Object::has_extension('MyExtension');
++		$foo = SS_Object::has_extension('MyExtension');
+	}
+}
+```
+
+If you are publishing a 3.x-compatible module that directly references to the `Object` class,
+it's a bit more complicated. You can apply one of two fixes:
+
+ * Mark your module as not working with PHP 7.2, by putting `"php": ">=5.3, <7.2"` in your `composer.json`.
+ * Create a new version of your module that requires `"silverstripe/framework": "^3.7"`.
+   In this version, replace all references to `Object` with `SS_Object`.
+   This version won’t work on SilverStripe 3.6,
+   but it will work on SilverStripe 3.7+ on any version of PHP.
```

Comment:
If module devs added the following one-liner to `_config.php`, would that mean they could support both 3.6 and 3.7 without having to create new release lines/tags?

```php
if (!class_exists('SS_Object')) class_alias('Object', 'SS_Object');
```

They could then swap out `Object::whatever()` for `SS_Object::whatever()` while still supporting 3.6. It won’t break anything on PHP 7.2, because `SS_Object` would already exist